src/assets/scripts/commandParser/commandMap.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * List of System Commands
+ *
+ * When a command is parsed, the value here will be used for the `name` property
+ * of the `CommandParser`
+ *
+ * @property SYSTEM_COMMANDS
+ * @type {Object}
+ * @final
+ */
+export const SYSTEM_COMMANDS = {
+    auto: 'auto',
+    clear: 'clear',
+    pause: 'pause',
+    tutorial: 'tutorial',
+    version: 'version',
+
+    '`': 'moveDataBlock',
+    airport: 'airport',
+    rate: 'rate',
+    timewarp: 'timewarp',
+    transmit: 'transmit'
+};
+
+/**
+ * Some commands are converted to unicode (to provide arrow characters) for specific shortkeys
+ *
+ * This maps those unicode values, converted to a string, to the correct root command
+ *
+ * @property UNICODE_COMMANDS
+ * @type {Object}
+ * @final
+ */
+const UNICODE_COMMANDS = {
+    '\\u2B61': 'altitude',
+    '\\u2B63': 'altitude',
+    '\\u2BA2': 'heading',
+    '\\u2BA3': 'heading'
```

Comment:
Thanks fair.  to be honest, I think inserting `t l` or `t r` on keypress will be the cleanest/easiest solution.  I thought of one other way to do it, but it's gross and very brittle.  If we programmatically add `t l` or `t r`, once the user presses enter it will get validated and parsed just like it would if the user typed it in.  I think this is going to be the best solution here.